internal/expr/prepare.go

```diff
@@ -306,5 +317,9 @@ func (pe *ParsedExpr) Prepare(args ...any) (expr *PreparedExpr, err error) {
 		}
 	}
 
-	return &PreparedExpr{inputs: inputs, outputs: outputs, sql: sql.String()}, nil
+	return &PreparedExpr{
+		inputs:  inputs,
+		outputs: outputs,
+		sql:     sql.String(),
+	}, nil
```

Comment:
Same comment as the previous PR. Let's please not establish this as a pattern.

Do you find it inconvenient to read these short one-liners?